lib/WOPI/DiscoveryManager.php

```diff
@@ -77,6 +77,35 @@ public function fetchFromRemote() {
 			$options['verify'] = false;
 		}
 
+		// first load with proxy can take some time - increase timeout in that case
+		$usesProxy = false;
```

Comment:
Would probably be nice to move this into a dedicated function so the general flow is a bit more readable and we might be able to reuse this in case we need it for any other request that might run into the default timeout.